engine/classes/ElggGroup.php

```diff
@@ -523,6 +523,12 @@ protected function create() {
 		global $CONFIG;
 		
 		$guid = parent::create();
+		if (!$guid) {
+			// @todo this probably means permission to create entity was denied
+			// Is returning false the correct thing to do
```

Comment:
I was having problems with saving a group and getting a Fatal Error. Due to permissions logic in another ticket, the group was never being saved, and this kept on trying to write to DB.